SignalMessaging/contacts/SystemContactsFetcher.swift

```diff
@@ -375,6 +378,7 @@ class SystemContactsFetcher: NSObject {
      *
      * @param   completion  completion handler is called on main thread.
      */
+    @objc
     public func requestOnce(completion completionParam: ((Error?) -> Void)?) {
         AssertIsOnMainThread()
```

Comment:
This method uses UIApplication, so this can't compile.

Reply:
Done.